<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KWinDecorationForm</class>
 <widget class="QWidget" name="KWinDecorationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>595</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,100,1">
   <item>
    <widget class="KLineEdit" name="searchEdit">
     <property name="clickMessage">
      <string>Search</string>
     </property>
     <property name="showClearButton" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDeclarativeView" name="decorationList">
     <property name="whatsThis">
      <string>Select the window decoration. This is the look and feel of both the window borders and the window handle.</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KPushButton" name="configureDecorationButton">
       <property name="text">
        <string>Configure Decoration...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="configureButtonsButton">
       <property name="text">
        <string>Configure Buttons...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="ghnsButton">
       <property name="text">
        <string>Get New Decorations...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QDeclarativeView</class>
   <extends>QGraphicsView</extends>
   <header>QtDeclarative/QDeclarativeView</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>searchEdit</tabstop>
  <tabstop>decorationList</tabstop>
  <tabstop>configureDecorationButton</tabstop>
  <tabstop>configureButtonsButton</tabstop>
  <tabstop>ghnsButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
